torchaudio/csrc/sox/utils.cpp

```diff
@@ -142,19 +146,24 @@ caffe2::TypeMeta get_dtype(
 
 caffe2::TypeMeta get_dtype_from_str(const std::string dtype) {
```

Comment:
I didn't see this function used anywhere, but I decided to change for completeness sake.

Reply:
Oh this is a helper function from removed feature, which we forgot to remove. Can you leave the function as-is, so that this PR contains only changes related to the 24-bit support. That way, if we need to revert the change, it is easier.